### ISSUE TITLE
Schedule job to every 2 weeks

### ISSUE THREAD

**Issue by sanks** [node-schedule/node-schedule#455]
Is that possible to schedule a job to be completed every 2 weeks on the same day of week and time?

**Comment by circa10a** [comment]
@sanks Yes, here's a sample cron schedule: https://crontab.guru/#30_1_1,15_*_*

`30 1 1,15 * *` will run at 01:00 on the 1st and 15th of every month

**Comment by jonaswide** [comment]
@circa10a Unfortunately, that is technically not every 2 weeks, that's just the 1st and 15th of a month. But the 15th plus 14 days will be the 29th, leaving the 30th and 31th to be skipped and therefore shifting the weekday.

I'm as well looking for the cron syntax for running a schedule every 2 weeks on the same day of the week ongoing.
